examples/applications/svm_gui.py

```diff
@@ -319,7 +319,13 @@ def get_parser():
 
 def main(argv):
     op = get_parser()
-    opts, args = op.parse_args(argv[1:])
+
```

Comment:
I'll revert back the change, but since the issue mentioned above by @jnothman is fixed now, any reason the tests are not passing?